db/schema.rb

```diff
@@ -49,7 +49,7 @@
     t.string   "name"
     t.decimal  "lat",             precision: 16, scale: 14, null: false
     t.decimal  "lng",             precision: 17, scale: 14, null: false
-    t.string   "city_id"
+    t.integer  "city_id"
```

Comment:
Not sure how this snuck in there -- you may need to run a `./bin/rake db:migrate` locally.